src/Confluent.RestClient/IConfluentClientSettings.cs

```diff
@@ -3,5 +3,6 @@
     public interface IConfluentClientSettings
     {
         string KafkaBaseUrl { get; }
+        long RequestTimeout { get; }
```

Comment:
Let's have the timeout as `Timespan` instead of long `TimeSpan RequestTimeout { get; }`

Thus, app config will have `<add key="Confluent.RequestTimeout" value="00:00:30"/>` and parse it to timespan.